resources/views/community/components/recent-awards/meta-type-filter.blade.php

```diff
@@ -0,0 +1,15 @@
+@props([
+    'selectedAwardType' => null,
+])
+
+<div class="grid gap-y-1 sm:pl-[30px] sm:pr-8">
+    <label class="text-xs font-bold" for="filter-by-award-kind">Award kind</label>
+
+    <select id="filter-by-award-kind" class="w-full" onchange="handleAwardTypeChanged(event)">
+        <option value="all" @if (!$selectedAwardType) selected @endif>All</option>
+        <option value="beaten-softcore" @if ($selectedAwardType === 'beaten-softcore') selected @endif>Beaten (softcore)</option>
+        <option value="beaten-hardcore" @if ($selectedAwardType === 'beaten-hardcore') selected @endif>Beaten</option>
+        <option value="completed" @if ($selectedAwardType === 'completed') selected @endif>Completed</option>
+        <option value="mastered" @if ($selectedAwardType === 'mastered') selected @endif>Mastered</option>
```

Comment:
The "Completed" filter also shows Masteries:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/806440d0-cabb-4295-8802-015151aee10a)

Similarly, the "Beaten (softcore)" filter also shows "Beaten" records:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0191abea-7549-45a5-add9-58702bc6a9f5)

"Mastered" and "Beaten" appear to filter correctly.

Reply:
Thanks, resolved in https://github.com/RetroAchievements/RAWeb/pull/1782/commits/17df9774823f44857764a86fdac723b87a8957c8.

Reply:
Now it seems like Mastered and Beaten are missing from All.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/635ce525-d6d2-49ab-8cb2-9f35d438476b)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/e3501a05-ea6b-499b-9c09-1ce5694bfbd6)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9eac497e-9f65-4153-b810-cb839ca000e4)

Reply:
Indeed. Resolved in https://github.com/RetroAchievements/RAWeb/pull/1782/commits/5fe3f5cb298a24a2b938d154ec7f2e25bf9c2feb.